test/test_config/initial_policy.yml

```diff
@@ -1 +1,3 @@
 - !variable one/password
+- !variable two/username
+- !variable three/secret
```

Comment:
it is yaml. Needs a new line
As we spoke about, I wouldnt touch the initial policy because it impacts other tests

Reply:
I have fixed the list test to overcome this problem